docs/eventlisteners.md

```diff
@@ -41,6 +41,38 @@ the following fields:
 [kubernetes-overview]:
   https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
 
+### ServiceAccountName
```

Comment:
I moved this section up since we now refer to it when discussing the trigger sa override in the next section